Upstream patch fixed in Icehouse-1

The following upstream patch was cherrypicked out of the following
commit and is required for a subsequent patch to apply cleanly.

commit 0f6f386c74754bc2e94a177d00d74bc074eba2fd
Author: Dolph Mathews <dolph.mathews@gmail.com>
Date:   Thu May 23 15:09:14 2013 -0500

    import only modules (flake8 H302)
    
    Change-Id: I0fa6fc6bf9d51b60fa987a0040168f3f0ef78a4a

diff --git a/keystone/middleware/ec2_token.py b/keystone/middleware/ec2_token.py
index daac10a..7cd007c 100644
--- a/keystone/middleware/ec2_token.py
+++ b/keystone/middleware/ec2_token.py
@@ -22,7 +22,7 @@ Starting point for routing EC2 requests.
 
 """
 
-from urlparse import urlparse
+import urlparse
 
 from eventlet.green import httplib
 import webob.dec
@@ -73,7 +73,7 @@ class EC2Token(wsgi.Middleware):
         # Disable 'has no x member' pylint error
         # for httplib and urlparse
         # pylint: disable-msg=E1101
-        o = urlparse(FLAGS.keystone_ec2_url)
+        o = urlparse.urlparse(FLAGS.keystone_ec2_url)
         if o.scheme == 'http':
             conn = httplib.HTTPConnection(o.netloc)
         else:
